pkg/processor/trigger/http/trigger.go

```diff
@@ -211,7 +213,75 @@ func (h *http) AllocateWorkerAndSubmitEvent(ctx *fasthttp.RequestCtx,
 	return response, false, nil, processError
 }
 
-func (h *http) requestHandler(ctx *fasthttp.RequestCtx) {
+func (h *http) handleRequest() fasthttp.RequestHandler {
+	return func(ctx *fasthttp.RequestCtx) {
+		if h.configuration.CORS.Enabled &&
+			common.ByteSlice2String(ctx.Method()) == h.configuration.CORS.PreflightRequestMethod {
+
+			// when CORS is enabled, processor HTTP server is responding to "PreflightRequestMethod" (e.g.: OPTIONS)
+			// That means => function will not be able to answer on the method configured by PreflightRequestMethod
+			h.handlePreflightRequest(ctx)
+		} else {
+			h.handleActualRequest(ctx)
+		}
+	}
+}
+
+func (h *http) handlePreflightRequest(ctx *fasthttp.RequestCtx) {
+
+	// default to bad preflight request unless all specifications are valid
```

Comment:
Suggest adding counters so we can account for preflight requests (without this, we'll see trigger counters rise but not handler counters).